src/todotxt/task.rs

```diff
@@ -22,6 +40,15 @@ pub enum CompletionMode {
     PriorityToTag,
 }
 
+/// How to set completion date on task completion.
+#[derive(PartialEq, Debug, Clone, Copy)]
+pub enum CompletionDateMode {
+    /// Only add completion date if task has create date is specified.
+    WhenCreateDateIsPresent,
+    /// Always add completion date, regardless of whether or not create date is present
```

Comment:
Here and above:  please note that it is "creation date" not "create date". Through the code and docs `creation date` is used. Though, once I also made the same typo - I have just found it - I'll fix it later 😄

Reply:
I was confused by field name being `create_date`, but after taking a closer look at yours and todotxt's documentation I agree - let's try keep these terms consistent